kview/src/main/java/dev/ikm/komet/kview/mvvm/view/genediting/PropertiesController.java

```diff
@@ -105,25 +108,31 @@ private void setupShowingPanelHandlers() {
 
         editFieldsJfxNode = FXMLMvvmLoader.make(config);
 
-        Config closePropertiesConfig = new Config(this.getClass().getResource("close-properties.fxml"))
-                .addNamedViewModel(new NamedVm("genEditingViewModel", genEditingViewModel));
-        JFXNode<Pane, ClosePropertiesController> closePropsJfxNode = FXMLMvvmLoader.make(closePropertiesConfig);
+        Config closePropertiesConfig = new Config(this.getClass().getResource(ConfirmationPaneController.FXML_FILE));
+        JFXNode<Pane, ConfirmationPaneController> closePropsJfxNode = FXMLMvvmLoader.make(closePropertiesConfig);
         closePropsPane = closePropsJfxNode.node();
-        closePropertiesController = closePropsJfxNode.controller();
+
+        ConfirmationPaneViewModel confirmationViewModel = (ConfirmationPaneViewModel) closePropsJfxNode
```

Comment:
Can you obtain object as a `ViewModel`?
```java
ViewModel confirmationViewModel = closePropsJfxNode.getViewModel(...).get();
```
Reason: avoid down casting.

Or you could do the following (defensive coding):
```java
Optional<ConfirmationPaneViewModel> confirmationViewModelOpt = closePropsJfxNode.getViewModel(...);
assert(confirmationViewModelOpt.isNotNull(), "Controller must contain a view model named " + ...);
confirmationViewModel = confirmationViewModelOpt.get();
```